content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -55,7 +55,7 @@ to use this feature with Kubernetes stateless pods:
 * CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
 Please note that containerd v1.7 supports user namespaces for containers,
-compatible with Kubernetes {{< skew currentVersion >}}. It should not be used
+compatible with Kubernetes {{< skew currentPatchVersion >}}. It should not be used
 with Kubernetes 1.27 (and later).
```

Comment:
Erm, I think we were supposed to change this page for the v1.27 release, and forgot to!